hack/vendor.sh

```diff
@@ -20,18 +20,20 @@ clone git github.com/tchap/go-patricia v2.1.0
 clone git golang.org/x/net 3cffabab72adf04f8e3b01c5baf775361837b5fe https://github.com/golang/net.git
 
 #get libnetwork packages
-clone git github.com/docker/libnetwork 0521fe53fc3e7d4e7c2e463800f36662c9169f20
+clone git github.com/docker/libnetwork dd1c5f0ffe7697f75a82bd8e4bbd6f225ec5e383
```

Comment:
@tiborvass This includes a fix for `etcd` (see [this comment](https://github.com/docker/docker/pull/16863#issuecomment-147682818))